### PR TITLE
docs: 7.4 what's new (Add expressions note)

### PR DESCRIPTION
**Pull request by kylebrandt**
add a note that expressions do not work with current alerting.

### CODE REVIEW

**Comment by kylebrandt** [comment]
FWIW, the Alerting UI will error if you attempt this:

image

But seemed to me worth calling out. If not here will capture it in the docs (If you think that is better, we can close this).

File: docs/sources/whatsnew/whats-new-in-v7-4.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ _Server-side expressions_ is an experimental feature that allows you to manipula
 
 The main use case is for [multi-dimensional](https://grafana.com/docs/grafana/latest/getting-started/timeseries-dimensions/#time-series-dimensions) data sources used with the upcoming next generation alerting, but expressions can be used with backend data sources and visualization as well.
 
-> **Note:** Queries built with this feature may break with minor version upgrades until Grafana 8 is released.
+> **Note:** Queries built with this feature may break with minor version upgrades until Grafana 8 is released. This feature also does not work with current Grafana alerting.
```
Comment (achatterjee-grafana) [inline docs/sources/whatsnew/whats-new-in-v7-4.md:78]:
```suggestion
> **Note:** Queries built with this feature may break with minor version upgrades until Grafana 8 is released. This feature also does not work with the current Grafana alerting.
```

**Review by achatterjee-grafana — APPROVED** [review]
Added a minor copy-edit suggestion.

File: docs/sources/whatsnew/whats-new-in-v7-4.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ _Server-side expressions_ is an experimental feature that allows you to manipula
 
 The main use case is for [multi-dimensional](https://grafana.com/docs/grafana/latest/getting-started/timeseries-dimensions/#time-series-dimensions) data sources used with the upcoming next generation alerting, but expressions can be used with backend data sources and visualization as well.
 
-> **Note:** Queries built with this feature may break with minor version upgrades until Grafana 8 is released.
+> **Note:** Queries built with this feature may break with minor version upgrades until Grafana 8 is released. This feature also does not work with the current Grafana alerting.
```
Comment (oddlittlebird) [inline docs/sources/whatsnew/whats-new-in-v7-4.md:78]:
```suggestion
> **Note:** Queries built with this feature might break with minor version upgrades until Grafana 8 is released. This feature does not work with the current Grafana alerting.
```

**Comment by oddlittlebird** [comment]
Added a minor editing suggestion.

**Pull request merged**